pkg/config/config.go

```diff
@@ -23,6 +24,7 @@ type ServerConfig struct {
 	IP        string `yaml:"ip"`
 	Bridge    string `yaml:"bridge"`
 	LogLevel  string `yaml:"log-level"`
+	Key       string `yaml:"key" validate:"required"`
```

Comment:
Use validate.RegisterValidation  to validate key
Also other filed is required except web